src/test/resources/features/contact/UpdateContact.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Manage Tasks in Salesforce
+  Background: User is authenticated
+    Given a user sing in the web page SalesForce
+
+@Acceptance
+Scenario: Update a Contact
+Given User sets the lastName
```

Comment:
Given, When and Then should be indented in respect to Scenario alignment.
In similar fashion, Scenario should also be indented in respecto to Feature alignment.